docs/storage.md

```diff
@@ -179,6 +189,38 @@ Now you can create the SR where `server` is your mon ip.
 * You may specify `serverport` option if you are using any other port than 6789.
 :::
 
+### Ceph iSCSI gateway
+
+This is at this moment the only way to connect to Ceph with no modifications of dom0, it's possible to create multiple Ceph iSCSI gateways following this: https://docs.ceph.com/docs/master/rbd/iscsi-target-cli/
+
+For some reason the chap authentication between gwcli and Xen doesn't seem to be working, so it's recommended to disable it (in case you use no authentication a dedicated network for storage should be used to ensure some security).
```

Comment:
Here user probably meant XCP-ng here, not Xen.

Reply:
Replaced with xcp-ng.